.travis.yml

```diff
@@ -70,7 +70,7 @@ jobs:
       compiler: gcc
       cache: ccache
       # Ubuntu Bionic build prerequisites
-      env: CMAKEFLAGS_EXTRA="-DLOCALECOMPARE=ON"
+      env: CMAKEFLAGS_EXTRA="-DLOCALECOMPARE=ON -DFAAD=ON -DWAVPACK=ON -DMAD=ON -DMODPLUG=ON"
```

Comment:
Why did you change the Ubuntu build flags? I thought we were using FFMPEG now, not FAAD?

Reply:
These weren't actually changed: https://github.com/mixxxdj/mixxx/pull/2791/files#diff-354f30a63fb0907d4ad57269548329e3R20